lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -142,6 +142,7 @@ protected function doGetStats()
      */
     protected function getSerializerValue()
     {
+        // Only PHP Serialization is supported in HHVM
```

Comment:
Ok, so why is this check here then?  My severs are running HHVM 3.4.0 and 3.5.0 and neither of them seem to define HHVM_VERSION, so should this check be removed entirely and just `'return Redis::SERIALIZER_PHP;`?